onnxruntime/core/framework/data_types.cc

```diff
@@ -971,6 +971,72 @@ const std::vector<MLDataType>& DataTypeImpl::AllNumericTensorTypes() {
   return all_numeric_size_tensor_types;
 }
 
+const std::vector<MLDataType>& DataTypeImpl::AllFixedSizeTensorAndSequenceTensorTypes() {
+  static std::vector<MLDataType> all_fixed_size_tensor_and_sequence_types =
```

Comment:
Can we define these using the existing ones? May reduce the binary size growth a little and avoid potential issues with inconsistent updates.

e.g.

```
static std::vector<MLDataType> all_fixed_size_tensor_and_sequence_types =
            [](){
              auto t = AllFixedSizeTensorTypes(); 
              const auto& s = AllSequenceTensorTypes();
              t.insert(t.end(), s.begin(), s.end());
              return t;
            }();
```

Reply:
Fixed for this and for existing types